### PR TITLE
perfs live in elixir-omg

### PR DESCRIPTION
**Pull request by InoMurko**
elixir-omg hosts perf tests, we can use a docker image for that smoke test

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -448,57 +448,6 @@ jobs:
             circleci tests split ./tags.txt > /tmp/tests-to-run
             mix test --exclude test $(cat /tmp/tests-to-run)
 
-  test_docker_compose_performance:
```
Comment (ayrat555) [inline .circleci/config.yml:451]: will perf tests run only in elixir-omg?

Comment (InoMurko) [reply .circleci/config.yml:451]: yeah

**Review by ayrat555 — APPROVED** [review]

**Comment by ayrat555** [comment]
@InoMurko maybe that's a not good idea because we won't detect changes in response speed.

**Comment by InoMurko** [comment]
@ayrat555 are you confident that `smoke performance tests` (as they're in elixir-omg) would catch meaningful, reliable results in cirleci? or is it enough to run them in spinnaker for a longer time?

**Comment by ayrat555** [comment]
>  are you confident that smoke performance tests (as they're in elixir-omg) would catch meaningful, reliable results in cirleci?

During my experiments, they caught monitor failures when I increased metrics values.

> or is it enough to run them in spinnaker for a longer time?

I think both types of tests are useful.

**Comment by InoMurko** [comment]
`During my experiments, they caught monitor failures when I increased metrics values.`
What do you mean by monitor failures?
`I think both types of tests are useful.`

I still don't see why. Can you elaborate?

**Comment by ayrat555** [comment]
> If you want to catch performance regressions, you need consistently isolated environment, otherwise your thresholds might be affected by non-isolating issues 

Now we don't have such problems and circle ic perf tests catches regressions.

**Comment by InoMurko** [comment]
Show me 😁

**Comment by ayrat555** [comment]
https://app.circleci.com/pipelines/github/omgnetwork/elixir-omg/8939/workflows/bcebdc24-e205-4f32-b009-64455735beb2/jobs/87535

In this circleci ci run, I added sleeps so datadog monitors would fail

**Comment by InoMurko** [comment]
Wait, but this was testing perf code, not childchain performance regression, right?

**Comment by ayrat555** [comment]
do you mean the code that you deleted? 

I thought you'll add new tests which will run through a docker container. Datadog has a monitor for `/submit_block` endpoint https://app.datadoghq.com/monitors/25732222

**Comment by InoMurko** [comment]
do you have data that shows that running `perf` smoke tests in elixir-omg and in childchain is beneficial?
do they show *performance* regressions (not performance tests) in watcher, watcher info or childchain?

I have never seen data that would show that. why?
- because it's a smoke test and the sample rate is too short
- the execution environment is not isolated enough that the results could be consistent between runs

**Pull request merged**